tests/test_pretrained_clip.py

```diff
@@ -0,0 +1,85 @@
+import clip
+import torch
+
+import model.model as module_arch
+
+
+def test_pretrained_clip_same_as_ours():
```

Comment:
Think this name is a bit confusing since pretrained clip is also the name of our model, maybe `test_official_clip_same_as_ours` or `test_correct_model_implementation` to keep it more generic?